src/dojos/cd-dojo-list.vue

```diff
@@ -1,16 +1,80 @@
 <template>
   <div class="cd-dojo-list">
-    <dojo-list-item v-for="dojo in dojos" :key="dojo.id" :dojo="dojo"></dojo-list-item>
+    <dojo-list-item v-for="dojo in paginatedDojos" :key="dojo.id" :dojo="dojo"></dojo-list-item>
+    <pagination ref="pagination" for="dojo-pagination" :records="dojos.length" :per-page="dojosPerPage" count-text=""></pagination>
   </div>
 </template>
 <script>
+  import Vue from 'vue';
+  import { Pagination, PaginationEvent } from 'vue-pagination-2';
+  import { clone } from 'lodash';
   import DojoListItem from './cd-dojo-list-item';
+  import dojoPaginationStore from './dojo-pagination-store';
 
   export default {
     name: 'dojoList',
     props: ['dojos'],
+    data() {
+      return {
+        dojosPerPage: dojoPaginationStore.state.dojosPerPage,
+      };
+    },
     components: {
+      Pagination,
       DojoListItem,
     },
+    computed: {
+      currentPage() {
+        return dojoPaginationStore.state.page;
+      },
+      // dojos to be displayed on the current page
+      paginatedDojos() {
+        const begin = (this.currentPage - 1) * this.dojosPerPage;
+        const end = begin + this.dojosPerPage;
+        dojoPaginationStore.commit('setCount', this.dojos.slice(begin, end).length);
+        return this.dojos.slice(begin, end);
+      },
+      // the number of valid pages to display data on
+      numOfPages() {
+        return Math.ceil(this.dojos.length / this.dojosPerPage);
```

Comment:
You haven't found a way to get https://github.com/matfish2/vue-pagination-2#computed-properties -> totalPages to work?